projects/1.20-fabric/assets/furnection/ls_furniture/lang/zh_cn.json

```
@@ -0,0 +1,750 @@
+
+  "config.ls_furniture.preview": "预览",
+  "config.ls_furniture.preview.tooltip": "显示或隐藏预览台",
+
+  "config.ls_furniture.search_mode.off": "关闭",
+  "config.ls_furniture.search_mode.on": "开启",
+  "config.ls_furniture.search_mode.automatic": "自动",
+  "config.ls_furniture.search_mode.dynamic": "动态",
+
+  "container.ls_furniture.workstation.config": "打开工作站设置",
+  "container.ls_furniture.workstation.preview.open": "开启预览",
+  "container.ls_furniture.workstation.preview.close": "关闭预览",
+  "container.ls_furniture.workstation.preview.title": "预览",
+  "container.ls_furniture.workstation.no_preview.title": "无可获取预览",
+  "container.ls_furniture.workstation.single_block": "单块",
+  "container.ls_furniture.workstation.l_shape_block": "L形堆叠",
+  "container.ls_furniture.workstation.vertical_block": "垂直堆叠",
+  "container.ls_furniture.workstation.horizontal_block": "水平堆叠",
+  "container.ls_furniture.workstation.two_by_two_block": "平面堆叠",
+
+  "container.ls_furniture.workstation": "工作站",
+  "container.ls_furniture.drawer": "抽屉",
+  "container.ls_furniture.mailbox": "邮箱",
+  "container.ls_furniture.counter_cupboard": "厨柜台",
+  "container.ls_furniture.counter_drawer": "抽屉柜台",
+  "container.ls_furniture.cabinet_cupboard": "壁挂厨柜台",
+  "container.ls_furniture.oven": "烤箱",
+  "container.ls_furniture.wardrobe": "衣柜",
+  "container.ls_furniture.fridge": "冰箱",
+  "container.ls_furniture.freezer": "冰柜",
+  "container.ls_furniture.crate": "板条箱",
+  "container.ls_furniture.tombstone.edit": "编辑墓碑信息",
+
+  "container.ls_furniture.oak_chest": "橡木箱子",
+  "container.ls_furniture.oak_chestDouble": "大型橡木箱子",
+  "container.ls_furniture.spruce_chest": "云杉木箱子",
+  "container.ls_furniture.spruce_chestDouble": "大型云杉木箱子",
+  "container.ls_furniture.birch_chest": "白桦木箱子",
+  "container.ls_furniture.birch_chestDouble": "大型白桦木箱子",
+  "container.ls_furniture.jungle_chest": "丛林木箱子",
+  "container.ls_furniture.jungle_chestDouble": "大型丛林木箱子",
+  "container.ls_furniture.acacia_chest": "金合欢木箱子",
+  "container.ls_furniture.acacia_chestDouble": "大型金合欢木箱子",
+  "container.ls_furniture.dark_oak_chest": "深色橡木箱子",
+  "container.ls_furniture.dark_oak_chestDouble": "大型深色橡木箱子",
+  "container.ls_furniture.mangrove_chest": "红树木箱子",
+  "container.ls_furniture.mangrove_chestDouble": "大型红树林木箱子",
+  "container.ls_furniture.cherry_chest": "樱花木箱子",
+  "container.ls_furniture.cherry_chestDouble": "大型樱花木箱子",
+  "container.ls_furniture.bamboo_chest": "竹箱子",
+  "container.ls_furniture.bamboo_chestDouble": "大型竹箱子",
+  "container.ls_furniture.crimson_chest": "绯红木箱子",
+  "container.ls_furniture.crimson_chestDouble": "大型绯红木箱子",
+  "container.ls_furniture.warped_chest": "绯红木箱子",
+  "container.ls_furniture.warped_chestDouble": "大型诡异木箱子",
+
+
+  "subtitles.block.workstation.ui.take_result": "工作站：合成",
+  "subtitles.item.tool.use": "工具：已使用",
+  "subtitles.block.blinds.use": "百叶窗：切换",
+  "subtitles.block.drawer.close": "抽屉：关闭",
+  "subtitles.block.drawer.open": "抽屉：开启",
+  "subtitles.block.chair-stool.untuck": "椅子：解开",

```

```suggestion
  "subtitles.block.chair-stool.untuck": "椅子：拉出",
```
这个字幕和下方字幕是在什么时候出现，建议测试